public/templates/header.php

```diff
@@ -1,13 +1,21 @@
+<?php
+require_once(dirname(__FILE__)."/../../common.php");
+require_once(dirname(__FILE__)."/../../modules/models/user.php");
+
+maybe_session_start();
+verify_logged_in();
+?>
+
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="x-ua-compatible" content="ie=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <script type="text/javascript" src="https://code.jquery.com/jquery-3.6.0.slim.min.js" integrity="sha256-u7e5khyithlIdTpu22PHhENmPcRdFiHRjhAuHcs05RI=" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
```

Comment:
are we still using jquery?

Reply:
yeah i'll ditch this too, i can do the same thing without it i think